source/Junkership.js

```diff
@@ -87,11 +98,21 @@ export default class Junkership extends Pixi.Sprite {
         this.hitBox.y = this.y + 1
     }
 
-    shoot() {
-        game.addChild(new Projectile(
-            this.x + this.width,
-            this.y + this.height/2,
-            this))
+    fire(curXpos,curYpos,curShip,wOffset,hOffset) {
+        if(this.powerUp.curPower == "peaShoota") {
+            this.powerUp.peaShoota(curXpos,curYpos,curShip,wOffset,hOffset)
```

Comment:
Like mentioned earlier, you can just call `this.powerUp.peaShoota(this)`, and then have this pull out all the values, since you're going to be passing the entire Junkership anyway.